### DISCUSSION TITLE
How do I build a chunk of javascript via bleep's scalajs - as a step in a plugin to stitch together a local web application?

### DISCUSSION THREAD

**Discussion by dwalend** [oyvindberg/bleep#431] [Q&A]
How do I build a chunk of javascript via scalajs - as a step in a plugin to stitch together a local web application?

So far I've only gotten the javascript generated by `bleep run FrontEnd` (which errors because running it uses the dom). That's not a great option for a part of something bigger.

(I have been able to write a bleep script that uses `commands.compile()` , but I haven't found the part of bleep to call to generate the javascript. Compile seems to just make class files.)

**Comment by KristianAN** [comment]
This is an open issue, you can follow it here: https://github.com/oyvindberg/bleep/issues/384
